pyproject.toml

```diff
@@ -29,6 +29,12 @@ python = "^3.6"
 "aws-cdk.aws-iam" = {version="*", optional=true}
 "aws-cdk.aws-dynamodb" = {version="*", optional=true}
 "aws-cdk.aws-lambda" = {version="*", optional=true}
+"aws-cdk.aws-stepfunctions" = {version="*", optional=true}
```

Comment:
Can you please sort the package names alphabetically within this and the next block? It just makes it easier to scan the whole thing for a specific name.

Reply:
Fixed in 7927d78cd4f574a42f780e1ea72dd2d77a8a4bb5